### PR TITLE
WIP: [droid] get audio devices capabilities from API

### PR DESCRIPTION
**Pull request by koying**
As discussed in https://github.com/xbmc/xbmc/pull/12229

/cc @fritsch @MartijnKaijser

### CODE REVIEW

File: xbmc/cores/AudioEngine/Sinks/AESinkAUDIOTRACK.cpp [hunk 1]
```diff
@@ -59,6 +63,134 @@ bool CAESinkAUDIOTRACK::HasAmlHD()
   return ((CJNIAudioFormat::ENCODING_DOLBY_TRUEHD != -1) && (CJNIAudioFormat::ENCODING_DTS_HD != -1));
 }
 
+static void LogAudoDevices(const char* stage, const CJNIAudioDeviceInfos& devices)
```
Comment (hudokkow) [inline xbmc/cores/AudioEngine/Sinks/AESinkAUDIOTRACK.cpp]: typo s/Audo/Audio

**Comment by MartijnKaijser** [comment]
Any news on this?

**Comment by koying** [comment]
That one can be closed, I guess. 
@fritsch ended up doing it another way, correct @fritsch ?

**Comment by fritsch** [comment]
Yeah - currently we use a "brute-force" way :-). Whatever you like best. If the above method would be "as failsafe" let's do that instead - in general: it's easier to ask for forgiveness than for permission.

**Comment by MartijnKaijser** [comment]
@peak3d perhaps something of your interest?

**Comment by MartijnKaijser** [comment]
Replaced by new PR

**Pull request closed without merging**